lib/classes/asset.js

```diff
@@ -31,7 +31,7 @@ const Asset = class Asset {
 		this._size = -1;
 
 		this._integrity = "";
-		this._pathname = path.join("/", pathname);
+		this._pathname = path.join("/", pathname).replace(/\\/g, "/");
```

Comment:
This replacement should be refactored to a utility function and used wherever we do this replacement.